packages/compass-components/src/components/editor.tsx

```diff
@@ -51,6 +51,7 @@ type EditorProps = {
   options?: Omit<IAceOptions, 'readOnly'>;
   readOnly?: boolean;
   completer?: unknown;
+  dataTestId?: string;
```

Comment:
Do you mind changing this to `data-testid` similar to how we do it for some other components? I know that it's not consistent everywhere, but aligning on the prop name that would actually be used in the tests makes it much easier to discover

Reply:
ah yeah no that would be actually awesome